tasks/insights-client.yml

```
@@ -58,7 +58,9 @@
   changed_when: true
 
 - name: Configure remediation
-  when: rhc_insights.remediation | d("present") == "present"
+  when:
+    - ansible_distribution_version is version('8.4', '>=')
```

```suggestion
    - ansible_distribution_version is version("8.4", ">=")
```
prefer double quotes, or at least be consistent with other quoting in surrounding code